app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([member.get_rating() for member in team])
```

Comment:
no need to create list 
```suggestion
    return sum(member.get_rating() for member in team)
```